### ISSUE TITLE
Pin irc message or add in own list

### ISSUE THREAD

**Issue by Sonic3R** [thelounge/thelounge#4240]
### Feature Description
I might ask someone on channel regarding some useful tip and I would like to pin it in order to remember the answer without searching in whole history

**Comment by termermc** [comment]
I don't think this is something worthy of a feature. Just create a channel and paste it in there. I have a channel on my network called `#notes` and I use that for this purpose. You can do similar, just register a private channel on a network and use that.

**Comment by el-profesor926** [comment]
Wouldn't you just copy and paste into your own notes somewhere?

**Comment by Warr1024** [comment]
- Being able to navigate to the original message in history, along with the context in which it happened, would be useful and not easily done by just copying and pasting into another channel (especially when you can't know ahead of time exactly how MUCH of the context you'll need).
- Users should probably not have to rely on IRC networks allowing you to create a channel just for this purpose, or even being able to self-query.
- Copy-paste works really badly for some users, esp. mobile users and there's no reason this needs to be more complex than just tapping a ⭐ button or something.
- Being able to remove messages from a favorites list (e.g. if it was superseded by a later update, or if it was actionable but now completed, or no longer relevant) would be really nice.
